setup.py

```diff
@@ -0,0 +1,32 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='pyglidein',
+    use_scm_version=True,
+    setup_requires=['setuptools_scm'],
+    description='Some python scripts to launch HTCondor glideins',
+    url='https://github.com/WIPACrepo/pyglidein',
+    author='WIPAC',
+    author_email='contact-us@icecube.wisc.edu',
+    classifiers=[
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+    ],
+    keywords='htcondor htc glidein',
```

Comment:
Is this htc, or dhtc?  Maybe better to just say "distributed computing"

Reply:
dHTC

Reply:
Updating to dHTC.